src/views/partials/nav/top.njk

```diff
@@ -0,0 +1,16 @@
+<nav class="govuk-language-select" role="navigation" aria-label="{{ lang.language_switcher }}">
```

Comment:
Whenever we make calls to the getTranslationsForView it retrieves the translations for the template requested and it also includes common / template translations for things like the header / footer. You might need to add add a translation in common.json for language_switcher.

Reply:
Good spot, checked the HTML and it was indeed missing! Have added this in https://github.com/companieshouse/acsp-manage-users-web/pull/66/commits/6daf44d8213e6a4192dce51edcac3eaa8fecd095